osu.Game/Overlays/RankingsOverlay.cs

```diff
@@ -135,6 +135,9 @@ public void ShowCountry(Country requested)
 
         private void loadNewContent()
         {
+            if (ruleset.Value == null)
+                return;
```

Comment:
When can this be the case?

Reply:
For some reason ruleset is null on first `Show()` call. I guess, not the best way to fix that - just first thought.

Reply:
Even if this is the correct fix for this, "for some reason" does not inspire much confidence. The proper way to fix code issues is to investigate, understand and *then* settle on a fix.

Reply:
It's pretty obvious why this is happening from a 5 second glance at the code. Even if you can't see it by looking at the code, one breakpoint should be enough to discern what's going on.